src/app/(dashboard)/projects/[projectId]/page.tsx

```diff
@@ -0,0 +1,307 @@
+"use client";
+
+import axios from "axios";
+import React from "react";
+
+import { UserCredDto } from "@/app/api/credmanager/route";
+import BreadCrumb from "@/components/breadcrumbs";
+import { Heading } from "@/components/ui/heading";
+import { Separator } from "@/components/ui/separator";
+import { Team, ContributionCalculation } from "@prisma/client";
+import { useSession } from "next-auth/react";
+import { useEffect, useState } from "react";
+import { CalculationResult } from "../../gitrepo/calculationResults";
+import { AnimatedTooltip } from "@/components/ui/animated-tooltip";
+import { TeamCalculationCreated } from "@/components/teams/teamCalculationCreated";
+import { Button } from "@/components/ui/button";
+import { useRouter } from "next/navigation";
+import { OverviewDto } from "@/app/api/teams/types/overview.dto";
+import { LoadingCircle } from "@/components/navigation/loading";
+import { TextGenerateEffect } from "@/components/ui/text-generate-effect";
+import { TeamInsights } from "../../teams/[teamId]/teamInsights";
+import { CalculationIntervalChart } from "../../teams/[teamId]/calculationIntervalChart";
+import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
+import { Tabs, TabsContent, TabsList, TabsTrigger } from "@/components/ui/tabs";
+import { TeamContributionTable } from "../../teams/[teamId]/teamContributionTable";
+import { GenerateAttestationModal } from "../../attestation/attestationModal";
+
+interface PageProps {
+  params: { projectId: string };
+}
+
+type UserTooltipDto = {
+  id: number;
+  userName: string;
+  totalCred: number;
+  type: string;
+};
+
+export default function TeamDetailsPage({ params }: PageProps) {
+  const teamId = params.projectId;
+  const breadcrumbItems = [
+    { title: "Projects", link: "/projects" },
+    { title: "Project details", link: `/projects/${teamId}` },
+  ];
+  const { data: session } = useSession();
+
+  const [isLoading, setIsLoading] = useState(true);
+  const [emptyTeam, setEmptyTeam] = useState(false);
+  const [team, setTeam] = useState<Team>();
+  const [overview, setOverview] = useState<OverviewDto>();
+  const [userCredDtos, setUserCredDtos] = useState<UserCredDto[]>([]);
+  const [registeredGitRepos, setRegisteredGitRepos] = useState([]);
+  const [userTooltipDto, setUserTooltipDto] = useState<UserTooltipDto[]>([]);
+  const [hasContributionRequest, setHasContributionRequest] = useState(false);
+  const [contributionCalculation, setContributionCalculation] =
+    useState<ContributionCalculation>();
+  const [pollingCount, setPollingCount] = useState<number>(0);
+
+  const router = useRouter();
+
+  useEffect(() => {
+    setIsLoading(true);
```

Comment:
This is not needed right? The default value is already true

Reply:
Hmm good point, let me test it out